App/Comments/CommentTableViewCell.swift

```diff
@@ -47,7 +48,7 @@ class CommentTableViewCell: SwipeTableViewCell {
         leftPaddingConstraint.constant = padding
     }
 
-    func updateCommentContent(with comment: Comment, theme: AppTheme) {
+    func updateCommentContent(with comment: Comment, theme: AppTheme, isPostAuthor: Bool? = nil) {
```

Comment:
Is the `isPostAuthor` param better as a non-nullable bool with default value of false?

Reply:
There may be a way we can side-step needing it to be optional if we make some larger changes, but the reason I made it optional was because in `SwipeCellKitActions` we call `cell.updateCommentContent(with: comment, theme: self.themeProvider.currentTheme)`. If `isPostAuthor` is defaulted to `false` then it would always be set to `false` after the user swipes on a cell. There is no context of the post or comment from within `SwipeCellKitActions` so there is no good way to overcome that.

Reply:
Good point, in fact it's probably something that HackersKit should figure out as part of the `Comment` model. Happy with your change and I'll add it to the HackersKit backlog.